src/main/java/phone/Call.java

```diff
@@ -0,0 +1,10 @@
+package phone;
+
+public class Call {
+    String callTime;
```

Comment:
there is a special object called LocalDateTime for storing time, ask me in class